### PR TITLE
Upgrade sbt typescript

### PR DESCRIPTION
**Pull request by wassertim**
@SadatAnwar sbt-typescript is ready :) so I bumped it up

### CODE REVIEW

**Comment by SadatAnwar** [comment]
Sounds cool! What about our tests? You can fix them?

**Comment by wassertim** [comment]
That's what I'm going to do next. I want to integrate karma runner into sbt test. So we will never need npm. Do we have Travis sbt build already?

Tim

**Comment by SadatAnwar** [comment]
I just looked at the travis error and it says 

`error TS5057: Cannot find a tsconfig.json file at the specified directory: './'`

Any idea why it would say that?

File: build.sbt [hunk 1]
```diff
@@ -31,31 +31,6 @@ libraryDependencies ++= Seq(
   "org.webjars.npm" % "bootstrap-sass" % "3.3.6"
 )
 
-TypescriptKeys.moduleKind := "commonjs"
```
Comment (SadatAnwar) [inline build.sbt]: All of this is now automatically done?

**Comment by wassertim** [comment]
Yes, I know what went wrong. I’ll fix that later :)

**Comment by SadatAnwar** [comment]
@wassertim could you please fix this? Or let me know what went wrong, and Ill do it :) I am eagerly waiting for the red to become green

**Comment by wassertim** [comment]
I close this, because I gave up :( see the new solution which I don't like: https://github.com/SadatAnwar/lunch-mator/pull/41

**Pull request closed without merging**